cmd/shellExecute_test.go

```diff
@@ -121,7 +121,7 @@ func TestRunShellExecute(t *testing.T) {
 	t.Run("success case - multiple positional script arguments gets added to the correct script", func(t *testing.T) {
```

Comment:
Should we include another test for the case where script1.sh has 0 arguments , while script2.sh has 1 or 2 arguments? 

Also isArgumentAtPosition does not consider any extra arguments users might have added by mistake , not sure if that could be a test as well